taxi/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import CharField
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> CharField:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=20, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer, on_delete=models.CASCADE, related_name="cars"
+    )
+    drivers = models.ManyToManyField(Driver, related_name="cars")
+
+    def __str__(self) -> CharField:

```

Comment:
The return type annotation for the `__str__` method should be `str`, not `CharField`. The `__str__` method returns a string representation of the object.